omod/src/main/webapp/fragments/conditionlist/conditions.gsp

```diff
@@ -44,7 +44,7 @@
         </ul>
     </div>
 
-    <div id="remove-condition-dialog" class="dialog" style="display: none; position: absolute; left: 35%; top:30%;">
+    <div id="remove-condition-dialog" class="dialog" style="display: none; position: absolute; left: 35%; top:30%;z-index: 1000 !important;">
```

Comment:
Can you improve the formatting by putting space where needed?

Reply:
Alright then I will